crates/model/src/trade.rs

```diff
@@ -2,15 +2,15 @@
 //! as described by the openapi documentation.
 
 use {
-    crate::order::OrderUid,
+    crate::{fee_policy::FeePolicy, order::OrderUid},
     num::BigUint,
     primitive_types::{H160, H256},
     serde::{Deserialize, Serialize},
     serde_with::{serde_as, DisplayFromStr},
 };
 
 #[serde_as]
-#[derive(Eq, PartialEq, Clone, Debug, Default, Deserialize, Serialize)]
+#[derive(PartialEq, Clone, Debug, Default, Deserialize, Serialize)]
```

Comment:
`Eq` was removed since FeePolicy contains `f64` types that don't derive this trait. Also, couldn't find a reason why this was required.